migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
To ensure folks aren't confused or run into issues, I think we want to remove the whole migrations folder and any contents since students should be generating their own migrations as they build out their models.